packages/components/Button/src/Button.settings.win32.ts

```diff
@@ -0,0 +1,75 @@
+import { buttonName, IButtonType } from './Button.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+
+export const settings: IComposeSettings<IButtonType> = [
```

Comment:
I had suggested this since alias tokens are only available for win32 (and I guess the default theme, which I forgot about). Since the styling wouldn't necessarily be correct for other platforms, it made sense to isolate the alias tokens to only the platform where they're guaranteed to work.

Reply:
@rurikoaraki It's hard for me to diff what changed between this win32.settings file and the baseline one, but I wonder if it'd be better practice to make those changes in a platform specific theme (either win32 or default) and leave the settings as is.  If there's something that we can't configure through tokens that needs to be different per platform we should probably put it on the tokens interface for control.

Reply:
@FalseLobster The main diff should be the color tokens (@ValentinaKozlova if there's other differences, feel free to chime in). We should be able to remove the split once we have other platforms supporting the vNext tokens, so this should be short lived.

Actually the default theme should be filled out in the next week or so, so if that's enough "support" we can get rid of this split then.

Reply:
There's were also changes in paddings/margins. Ok, I'm confused with Button.settings.ts and Button.settings.win32.ts. Should I revert my changes from Button.settings.ts? Shoud we have this split or not? Also, if this Button will be deprecated, maybe I had to change ExperimentalButton?

Reply:
I think this is fine - if the padding/margins align between default and win32, it'll be easier to align them later.

I thought the issue with using experimental button was that you couldn't import it into the menu button, because the menu button is under the components directory? Sure it would be preferable to use experimental button, but I thought that wouldn't be feasible without recreating the menu button under experimental? This Button is the only one we export externally so it's not deprecated yet.

The split should last until I get the alias tokens working for the default theme and can ensure they show up properly on other platforms. So I think we should keep the split, but it shouldn't last longer than a few months. If people feel strongly about having split styles, I can look into some temp alternatives on the theming side.

Reply:
Yes, there was an issue with using experimental Button. We agreed with Alicia that I would build MenuButton also for the new framework.